CHANGELOG.md

```diff
@@ -24,9 +24,63 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### [shopsys/framework]
+#### Changed
+- [#389 - Consistent Yaml indentation size](https://github.com/shopsys/shopsys/pull/389)
+    - standardized yaml files indentation to 4 spaces
+
 #### Fixed
 - [#260 - JS validation: dynamically added form inputs are now validated](https://github.com/shopsys/shopsys/pull/260)
 
+### [shopsys/framework]
+#### Changed
+- [#389 - Consistent Yaml indentation size](https://github.com/shopsys/shopsys/pull/389)
+    - standardized yaml files indentation to 4 spaces
+
+### [shopsys/coding-standards]
+#### Changed
+- [#389 - Consistent Yaml indentation size](https://github.com/shopsys/shopsys/pull/389)
+    - standardized yaml files indentation to 4 spaces
+
+### [shopsys/http-smoke-testing]
+#### Changed
+- [#389 - Consistent Yaml indentation size](https://github.com/shopsys/shopsys/pull/389)
+    - standardized yaml files indentation to 4 spaces
+
+### [shopsys/migrations]
+#### Changed
+- [#389 - Consistent Yaml indentation size](https://github.com/shopsys/shopsys/pull/389)
+    - standardized yaml files indentation to 4 spaces
+
+### [shopsys/plugin-interface]
+#### Changed
+- [#389 - Consistent Yaml indentation size](https://github.com/shopsys/shopsys/pull/389)
+    - standardized yaml files indentation to 4 spaces
+
+### [shopsys/product-feed-google]
+#### Changed
+- [#389 - Consistent Yaml indentation size](https://github.com/shopsys/shopsys/pull/389)
+    - standardized yaml files indentation to 4 spaces
+
+### [shopsys/product-feed-heureka]
+#### Changed
+- [#389 - Consistent Yaml indentation size](https://github.com/shopsys/shopsys/pull/389)
+    - standardized yaml files indentation to 4 spaces
+
+### [shopsys/product-feed-heureka-delivery]
+#### Changed
+- [#389 - Consistent Yaml indentation size](https://github.com/shopsys/shopsys/pull/389)
+    - standardized yaml files indentation to 4 spaces
+
+### [shopsys/product-feed-zbozi]
+#### Changed
+- [#389 - Consistent Yaml indentation size](https://github.com/shopsys/shopsys/pull/389)
+    - standardized yaml files indentation to 4 spaces
+
+### [shopsys/project-base]
+#### Changed
+- [#389 - Consistent Yaml indentation size](https://github.com/shopsys/shopsys/pull/389)
+    - standardized yaml files indentation to 4 spaces
```

Comment:
I consider having the same message about whitespace changes copy-pasted in the CHANGELOG.md eleven times ridiculous, to be honest. Can we do it some other way? Maybe mention it just in shopsys/shopsys?

Reply:
Well, this is a valid point, but copy-paste is the way we do it when the change includes more packages. Try to run `cat CHANGELOG.md | sort` and You'll see it is normal